src/lib/components/SgtSvgIcon/SgtSvgIcon.vue

```diff
@@ -59,6 +61,9 @@ export default class SgtSvgIcon extends Vue {
 </script>
 
 <style lang="scss">
+.sgt-icon-container {
+  display: inline-block;
+}
 .icon-container {
   display: inline-block;
   cursor: pointer;
```

Comment:
This style can be moved to the sgt-icon-container class

Reply:
Done